server/src/internalClusterTest/java/org/opensearch/search/nested/SimpleNestedExplainIT.java

```diff
@@ -70,7 +75,23 @@ public void testExplainMultipleDocs() throws Exception {
             .setRefreshPolicy(IMMEDIATE)
             .get();
 
-        indexRandomForMultipleSlices("test");
+        client().prepareIndex("test")
```

Comment:
@neetikasinghal apologies if that was asked, but why we need this doc? It is not searched against ...

Reply:
this is to ensure that 2 slices get created for concurrent search path to kick in and avoid the `indexRandomForMultipleSlices ` as that has deletion of bogus docs that messes up with the range in the explanation of the search results.